lib/plugins/aws/package/lib/mergeIamTemplates.js

```diff
@@ -69,6 +69,11 @@ module.exports = {
     );
     iamRoleLambdaExecutionTemplate.Properties.Path = this.provider.naming.getRolePath();
     iamRoleLambdaExecutionTemplate.Properties.RoleName = this.provider.naming.getRoleName();
+
+    if ('permissionsBoundary' in this.serverless.service.provider) {
```

Comment:
We should accepty only truthy values, so let's do:
```javascript
if (this.serverless.service.provider.permissionsBoundary) {
...
```

Reply:
@medikoo I have made the requested changes